mm2src/coins/lp_coins.rs

```diff
@@ -1925,12 +1930,29 @@ pub enum WithdrawError {
         available: BigDecimal,
         required: BigDecimal,
     },
+    #[display(fmt = "DB error {}", _0)]
+    DbError(String),
 }
 
 impl From<GetNftInfoError> for WithdrawError {
     fn from(e: GetNftInfoError) -> Self { WithdrawError::GetNftInfoError(e) }
 }
 
+impl<T: NftStorageError> From<T> for WithdrawError {
+    fn from(err: T) -> Self {
+        let msg = format!("{:?}", err);
+        WithdrawError::DbError(msg)
+    }
+}
+
+impl From<CreateNftStorageError> for WithdrawError {
+    fn from(e: CreateNftStorageError) -> Self {
+        match e {
+            CreateNftStorageError::Internal(err) => WithdrawError::InternalError(err),
+        }
+    }
+}
+
```

Comment:
I would like to not add coin/protocol specific things in `lp_coins` and keep it in a generic way. We should even move the other coin/protocol specific things from it, which we will do in future. So adding things like this will increase the tech debt more and more.

Reply:
Sorry, didnt get your idea about generics. lp_coins is a root module of coins lib, it will contain some coins specific things. in another way we need to completely refactor lp_coins and make more modules.

Reply:
What I meant is to keep `lp_coins` as the handler of `coins` crate in a generic way rather than having `nft`, `eth`, `tenderment` specifics in it. Because this makes having `eth`, `nft`, `tendermint`, etc. modules pointless. We should use the coin/protocol specifics in their own modules.

Reply:
Could you give a hint, do we have such refactoring issue, or we need to create the new one?

Reply:
Yeah yeah :D I just wanted to make a reference to your comment in refactoring issue if we have it

Reply:
It's part of the crate refactoring task which we initialized 5-6 months ago. Before that crates were coupled even more. But still, we need more work on this task. I don't know if we have issue created for this, but I know we all aware of it which is also a blocker for other tasks such as providing komodo sdk via C ABI. As far as I remember I already mentioned coins and protocols will be seperated crates. We don't want to compile whole coins crate for exposing tendermint protocol.

Reply:
moved nft error implementations into nft related mods